__tests__/app.test.js

```diff
@@ -12,6 +13,14 @@ afterAll(() => {
    return seed(data)
  });
 
+ describe('GET /api', () => {
+  test('200: Should return a JSON object with descriptions of all endpoints available', () => {
+    return request(app).get('/api').expect(200).then(({body}) => {
+      expect(body).toEqual(jsonEndPoints)
+    })
+  })
+ })
```

Comment:
This is a really nice test. By setting this up dynamically you're able to add to your json file and continue to pass this test 🤸